modules/ossm-threescale-integrate.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/threescale-adapter.adoc
+
+[id="ossm-threescale-integrate_{context}"]
+= Integrate the 3Scale adapter with {ProductName}
+You can use these examples to configure requests to your services using the 3scale Istio Adapter.
+
+
+.Prerequisites:
+
+* {ProductName} 0.7.0+
+* A working 3scale account (link:https://www.3scale.net/signup/[SaaS] or link:https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.5/html/installing_3scale/onpremises-installation[3scale 2.5 On-Premises])
+* xref:../service_mesh_install/preparing-ossm-installation.html#ossm-install-prerequisites_preparing-ossm-installation[{ProductName} prerequisites]
```

Comment:
I'd remove this li. It should be covered by the having Service Mesh installed.

Reply:
I'll leave that for @dfennessy and the 3scale team to determine.